### PR TITLE
chore(package.json): move angular to peerDependencies

### PR DESCRIPTION
**Pull request by Nightapes**
We have the issue with using the library:

```
ERROR in node_modules/@angular/core/src/render3/ng_dev_mode.d.ts(9,11): error TS2451: Cannot redeclare block-scoped variable 'ngDevMode'.
node_modules/@zxing/ngx-scanner/node_modules/@angular/core/src/render3/ng_dev_mode.d.ts(9,11): error TS2451: Cannot redeclare block-scoped variable 'ngDevMode'. 
 ```

I moved angular to peerDependencies

### CODE REVIEW

**Comment by odahcam** [comment]
So, I'm concerned about npm creating a node_modules folder inside the ngx-scanner instalation. Can you confirm that behavior?

**Comment by Nightapes** [comment]
Yes, i have a node_modules folder. npm installs the dependencies per module extra. So we should move the dependencies to peerDependencies.

**Comment by odahcam** [comment]
So moving @angular into peerDependencies would avoid an extra node_modules folder inside ngx-scanner package?

Just to confirm, I got some strange behavior in my last test, so I could not get to a conclusion.

**Comment by odahcam** [comment]
Okay, I'm just not sure that this works with Angular 2. 🤔 

After confirm that, we can finally merge.

**Comment by Nightapes** [comment]
It should still work with Angular 2, because the peerDependencies is  `">= 2.0.0"`

**Comment by odahcam** [comment]
I'm talking about the compatibility itself, I don't know if Angular 2 supports this lib anymore and I don't know if we should give support to Ng 2 at this time.

File: package.json [hunk 1]
```diff
@@ -81,9 +84,11 @@
         "karma-jasmine-html-reporter": "^0.2.2",
         "ng-packagr": "^2.2.0",
         "protractor": "^5.3.0",
+        "rxjs": "^5.5.7",
         "ts-node": "^4.1.0",
         "tslint": "^5.9.1",
-        "typescript": "2.6.2"
+        "typescript": "^2.6.2",
```
Comment (werthdavid) [inline package.json]: Is Angular CLI still giving warnings with typescript > 2.6.2?

**Pull request merged**